robots.txt

```diff
@@ -0,0 +1,4 @@
+user-agent: *
+disallow: /
+
+Sitemap: https://main--servicenow--hlxsites.hlx.live/blogs/sitemap-index.xml
```

Comment:
Can you please add to the go live checklist that you created that we will need to modify this robots.txt to put the servicenow domain and allow crawling?